fire/cli/mtl.py

```diff
@@ -585,12 +644,20 @@ def gama_beregning(
 
         # Observationer
         gamafil.write("\n\n<height-differences>\n\n")
-        for obs in observationer[["fra", "til", "dH", "L", "σ", "journal"]].values:
+        for obs in observationer[
+            ["fra", "til", "sluk", "dH", "L", "σ", "journal"]
+        ].values:
```

Comment:
Hvis du folder `obs` ud med det samme kan du få mere læsbar kode længere nede. Det er svært at lure hvad `obs[3]` dækker over, men lynhurtigt at forstå hvad `dH` betyder:

```suggestion
        for (fra, til, sluk, dH, L, σ, journal) in observationer[
            ["fra", "til", "sluk", "dH", "L", "σ", "journal"]
        ].values:
```

Reply:
ja - denne her har jeg selv krummet tæer over nogle gange undervejs. jeg tror faktisk det kan gøres endnu enklere, men jeg skal lige have checket op på Pandas' ekscentriciteter endnu en gang... mere senere

Reply:
Ingen stress, jeg ved jo godt det er work in progress. Jeg snublede bare lige over denne

Reply:
Rettet ved at lade `obs in observationer` være en named tuple